### ISSUE TITLE
GitLab Support

### ISSUE THREAD

**Issue by Yoshi325** [jasonbellamy/git-label#21]
@jasonbellamy how would you feel if I did a PR to add GitLab support to this?

**Comment by johannesschobel** [comment]
that would be awesome.. however, i feel like this should / could be a dedicated package? @Yoshi325 @jasonbellamy

**Comment by Yoshi325** [comment]
I don't think that is a bad idea. But given the name of this project (`git-label`, v. `github-label`), I thought it was best to start the discussion. If @jasonbellamy isn't open to a PR, or we don't hear from him for a while, I may just do exactly what you suggested @johannesschobel.

**Comment by johannesschobel** [comment]
I mean - another (possibly quite interesting approach) would be to have some kind of `driver()` method, where you can select `github`, `gitlab` or `bitbucket`.. or an additional `custom` where you need to provide specific connection properties (e.g, the serverhost, and so on)..
